src/index.html

```diff
@@ -4,9 +4,375 @@
+        </div>
+  
+        <div class="card__code">
+          Product code: 195434
+        </div>
+  
+        <div class="card__grade">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__grade-reviews">
+            Reviews: 5
+          </div>
+        </div>
+  
+        <div class="card__price-container">
+          <p class="card__price-text-paragraf">Price:</p>
+          <p class="card__price-paragraf">$2,199</p>
+        </div>
+  
+        <div class="card__buy" >
+          <a class="card__buy-link">BUY</a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          alt="imac"
+          src="images/imac.jpeg"
+        >
+  
+        <div class="card__link">
+          <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+        </div>
+  
+        <div class="card__code">
+          Product code: 195434
+        </div>
+  
+        <div class="card__grade">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__grade-reviews">
+            Reviews: 5
+          </div>
+        </div>
+  
+        <div class="card__price-container">
+          <p class="card__price-text-paragraf">Price:</p>
+          <p class="card__price-paragraf">$2,199</p>
+        </div>
+  
+        <div class="card__buy" >
+          <a class="card__buy-link">BUY</a>
+        </div>
+  
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          alt="imac"
+          src="images/imac.jpeg"
+        >
+  
+        <div class="card__link">
+          <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+        </div>
+  
+        <div class="card__code">
+          Product code: 195434
+        </div>
+  
+        <div class="card__grade">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__grade-reviews">
+            Reviews: 5
+          </div>
+        </div>
+  
+        <div class="card__price-container">
+          <p class="card__price-text-paragraf">Price:</p>
+          <p class="card__price-paragraf">$2,199</p>
+        </div>
+  
+        <div class="card__buy" >
+          <a class="card__buy-link">BUY</a>
+        </div>
```

Comment:
it's a redundant `div` wrapper
add href attribute to a link
```suggestion
          <a class="card__buy-link">BUY</a>
```

Reply:
Without "div" its dont work, i can use margin, because of BEM, padding dont work and make only button bigger. Plese help understand how i can do this, i try but its dont work.

Reply:
You can move `card__buy` styles to the link.

Reply:
But i already said "Padding" in link dont work. Button start only grow with padding, margin i cant use, becauce BEM.